invenio_rdm_records/theme/views.py

```diff
@@ -160,3 +124,11 @@ def vocabulary_subtitle(dict_key, vocabulary_key):
     vocabulary = Vocabularies.get_vocabulary(vocabulary_key)
```

Comment:
I think you can completely remove vocabulary_title and vocabulary_subtitle. Instead the values should be available in the serialised record. @Glignos was working on adding it i think.

Reply:
:+1: on this because `vocabulary.get_title_by_dict(dict_key).split('/')[1].strip()` raises an IndexError when the "Poster" resource type is selected for instance (`[1]` -> `[-1]` other solution, but using the serialized record value is even better if possible)..

Reply:
@fenekku there is a separate issue for that. https://github.com/inveniosoftware/invenio-app-rdm/issues/340 I was planning to address it separately since this PR is growing a bit too much.